sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -21,7 +21,7 @@
 == Notice
 
 [%hardbreaks]
-Copyright (C) 2021 Intel Corporation.  All rights reserved.
+Copyright (C) 2024 Intel Corporation.  All rights reserved.
```

Comment:
Our new specification only lists one date, which is the date of first publication.  Therefore, we should not update that date when making modifications.

Reply:
Oops, I thought the process was the other way around. Fixed in 5bec99fa3.

Reply:
I think you lost the fix that was in 5bec99fa3 when you force-pushed commits to this PR.

Reply:
I did!  Thanks for double-checking.  Fixed (again) in 974aec94af2ab81014895cf961895b5d2c06fc29.